### PR TITLE
Garagelimit

### PR DESCRIPTION
**Pull request by DardoTheMaster**
Hi all
As the previous pull request,these edits allow server admins to set a limit for the vehicles that each player is able to store. I'd like someone to test it (again) before an eventual merge.

Hope the feature will get appreciated!

### CODE REVIEW

**Comment by DardoTheMaster** [comment]
What is a tab error ? O_O

**Comment by Jawshy** [comment]
You need to use spaces - not tabs - as per https://github.com/ArmaLife/Framework/issues/646.

File: Altis_Life.Altis/stringtable.xml [hunk 1]
```diff
@@ -4434,6 +4439,10 @@
             <Russian>%2 получить обратно в черный список автомобиль возле %1.</Russian>
             <German>%2 hat ein gesuchtes Fahrzeug nahe %1 ausgeparkt.</German>
         </Key>
+        <Key ID="STR_COP_TooMuchVehicles">
+            <Original>The owner of this vehicle did not have space left in his garage.The vehicle got deleted.</Original>
```
Comment (sargken) [inline Altis_Life.Altis/stringtable.xml]: Space before the.

**Comment by DardoTheMaster** [comment]
Don't get the point of denying the use of tabs. Closing as I don't want to waste time on deleting tabs. SQF does not interpret spaces and tabs,so why would you?!

**Comment by BoGuu** [comment]
Please see the issue @Jawshy posted @DardoTheMaster 

Also, if you are struggling with the tabs, I will auto convert them over.

**Comment by berkaslan** [comment]
@DardoTheMaster pretty cool, but where need to put limit? :1234:

**Comment by DardoTheMaster** [comment]
@berkaslan on the cfgmaster,instead of the false value.

**Comment by DardoTheMaster** [comment]
@BoGuu My c9 editor seems to put tabs automatically when pushing on github. Couldn't find a work around. If you think the feature could be useful,i'd be grateful if you converted it for me.

**Comment by BoGuu** [comment]
@DardoTheMaster 

Make sure Soft Tabs is checked (and set to 4):

https://i.gyazo.com/8cbf747842dd2f829a65594eda0069c6.png

**Comment by DardoTheMaster** [comment]
Unfortunately,It was already set like this

File: Altis_Life.Altis/Config_Master.hpp [hunk 2]
```diff
@@ -115,7 +115,7 @@ class Life_Settings {
     vehicle_infiniteRepair = false; //Set to true for unlimited repairs with 1 toolkit. False will remove toolkit upon use.
     vehicleShop_rentalOnly[] = { "B_MRAP_01_hmg_F", "B_G_Offroad_01_armed_F", "B_Boat_Armed_01_minigun_F" }; //Vehicles that can only be rented and not purchased. (Last only for the session)
     vehicleShop_3D = false; //Add preview 3D inside Shop vehicle.       Default : False
-
+    garage_limit = false;
```
Comment (sargken) [inline Altis_Life.Altis/Config_Master.hpp]: That should be a integer not a boolean.

**Comment by DardoTheMaster** [comment]
@sargken  there's a check for this. Check the script

**Pull request closed without merging**